SerialUART/CPP/SerialSampleCpp.vcxproj

```diff
@@ -0,0 +1,194 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{b5690b48-a2fa-44f8-a7a4-18af7ea247e6}</ProjectGuid>
+    <RootNamespace>SerialSampleCpp</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>

```

Comment:
This was set to 19041 in other projects, why not this one?

Reply:
<TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>

Changed the Target PlatformVersion